File: Dataset/cs/Fullname.cs

//  Fullname.cs  (c) 2003 Kari Laitinen

using System ;

class Fullname
{
   static void Main()
   {
      string  first_name ;
      string  last_name ;

      Console.Write( "\n Please, type in your first name: " ) ;
      first_name  =  Console.ReadLine() ;
      Console.Write( "\n Please, type in your last name:  " ) ;
      last_name   =  Console.ReadLine() ;

      Console.Write( "\n Your full name is "
                  +  first_name  +  " "  +  last_name  +  ".\n" ) ;
   }
}

File: Dataset/cpp/fullname.cpp

//  fullname.cpp  (c) 1997-2002 Kari Laitinen

#include  <iostream.h>

int main()
{
   char  first_name[ 40 ] ;
   char  last_name [ 40 ] ;

   cout <<  "\n Please, type in your first name: " ;
   cin  >>  first_name ;
   cout <<  "\n Please, type in your last name:  " ;
   cin  >>  last_name ;

   cout <<  "\n Your full name is "
        <<  first_name  <<  " "  <<  last_name  <<  ".\n" ;
}


